### PR TITLE
chore: simplify runInstrumentedFunction deps

### PR DESCRIPTION
**Pull request by hazzadous**
We were passing in the hub which is a large dependency, just to use it
for statsd. We don't need statsd any more as there are prom metrics now.

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: plugin-server/src/main/utils.ts [hunk 1]
```diff
@@ -39,8 +35,6 @@ export async function runInstrumentedFunction<T, E>({
         Sentry.captureException(error, { tags: { team_id: teamId } })
         throw error
     } finally {
-        server.statsd?.increment(`${statsKey}_total`)
-        server.statsd?.timing(statsKey, timer)
```
Comment (tiina303) [inline plugin-server/src/main/utils.ts:43]: I don't think we have prom metrics for these, could you add them?

Comment (hazzadous) [reply plugin-server/src/main/utils.ts:43]: This is the end call above

Comment (hazzadous) [reply plugin-server/src/main/utils.ts:43]: Will update dashboards though

Comment (tiina303) [reply plugin-server/src/main/utils.ts:43]: can I get those dash updates? Time spent is quite a nice metric to use

**Pull request merged**